syntax = "proto3";

option java_package = "ex.grpc";

package logmessagetransporter;

service LogMessageTransporter {
    rpc sendLogMessages(MessagesRequest) returns (AcknowledgeReply) {
    }
}

// The log message container
message MessagesRequest {
    int32 size = 1;
    repeated string logMessages = 2;
}

// The acknowladge answer
message AcknowledgeReply {
    bool result = 1;
}